cmd/kots/cli/namespace.go

```diff
@@ -1,12 +1,39 @@
 package cli
 
 import (
+	"fmt"
 	"strings"
 
 	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/validation"
+	"k8s.io/client-go/tools/clientcmd"
 )
 
+func getNamespaceOrDefault(namespace string) (string, error) {
+	if namespace == "" {
+		clientCfg, err := clientcmd.NewDefaultClientConfigLoadingRules().Load()
+		if err != nil {
+			return "", errors.Wrap(err, "failed to load kubeconfig")
+		}
+		cctx := clientCfg.Contexts[clientCfg.CurrentContext]
+		if cctx != nil {
+			namespace = cctx.Namespace
+		}
+	}
+
+	if namespace == "" {
+		namespace = metav1.NamespaceDefault
+	}
+
+	err := validateNamespace(namespace)
+	if err != nil {
+		return "", errors.Wrap(err, fmt.Sprintf("invalid namespace %s", namespace))
```

Comment:
I think there's also an `errors.Wrapf(...)` function that does this, so we wouldn't even have to import `"fmt"` here.